server/files/stylesheets/docs.css

```diff
@@ -42,7 +42,6 @@ footer.footer {
   width: 0.7em;
   height: auto;
   vertical-align: middle;
```

Comment:
I think better use default `baseline` by removing this.
![fui-doc-twitter-logo-align](https://github.com/fomantic/Fomantic-UI-Docs/assets/930315/08a87bd2-e6e2-4a7c-83e3-9e2a21c11368)

Reply:
Yes it is looking good without using vertical-align.
I will change it

Reply:
Please add also `margin-left: 0.25em` back. I was wrong about that and already mentioned it.

![fui-doc-twitter-logo-update](https://github.com/fomantic/Fomantic-UI-Docs/assets/930315/4b44afc7-759e-4c88-a794-82e6f458760a)